tests/acceptance/pageObjects/phoenixPage.js

```diff
@@ -52,6 +52,28 @@ module.exports = {
       return this.waitForElementPresent('@messageCloseIcon')
         .click('@messageCloseIcon')
         .waitForElementNotPresent('@messageCloseIcon')
+    },
+    getNotifications: async function () {
+      const notifications = []
+      await this
+        .click('@notificationBell')
+        .waitForElementVisible('@notificationElement')
+      await this.api.elements('@notificationElement', result => {
+        for (let i = 0; i < result.value.length; i++) {
+          this.api.elementIdText(result.value[i].ELEMENT, text => {
+            notifications.push(text.value)
+          })
+        }
+      })
+      return notifications
+    },
+    assertNotificationIsPresent: async function (dataTableHashed) {
+      let isPresent
+      const notifications = await this.getNotifications()
+      for (const element in dataTableHashed) {
```

Comment:
Please use `for ... of` here instead of `for ... in` which will directly provide the elements inside the array.
```suggestion
      for (const element of expectedNotifications) {
```